editor/src/clj/editor/defold_project_search.clj

```diff
@@ -44,6 +44,7 @@
                                          search-value-fn (when (:search-fn resource-type)
                                                            (:search-value-fn resource-type))]
                                      (when (and search-value-fn
+                                                (resource/exists? resource)
```

Comment:
After deleting the file, node-id and resource continue to exist in the graph, but the resource file no longer exists on disk. `textual?` on the line below attempts to read from disk.